.github/workflows/sonar.yml

```diff
@@ -0,0 +1,44 @@
+name: SonarQube analysis
```

Comment:
As this repo is not in rootstrap's org you can delete this workflow so it doesn't run on every PR